### ISSUE TITLE
Add authentication for screenshots

### ISSUE THREAD

**Issue by tacman** [Spomky-Labs/phpwa#134]
### Description

If I want a screenshot of /my-profile, the user needs to be logged in.  

### Example

    bin/console pwa:create:screenshot https://127.0.0.1:8003/my-profile

![screenshot-1200x1100](https://github.com/Spomky-Labs/pwa-bundle/assets/619585/544a18dc-8b9c-4cba-9fbf-712815f80ccb)

I think panther has some support for this, but I don't really know.  Maybe 

    bin/console pwa:create:screenshot https://127.0.0.1:8003/my-profile --username=tac --password=my.password

**Comment by tacman** [comment]
As an alternative, how about the option to pass a header when taking the screenshot, which could include some sort of cookie or authentication.  It's messier, of course, but adding login is probably more complicated than it's worth.

If #138 worked, I'd probably take the screenshots locally and disable security, though that's not very feasible either.  Passing a header, though clunky, would work.

**Comment by Spomky** [comment]
Screenshot feature will be deprecated in 1.3.0.
This is a real PITA...

**Comment by tacman** [comment]
Makes sense, though I quite liked that feature!  But it's a lot of work for a PWA-specific bundle.
